### ISSUE TITLE
EL parser converts non-EL descriptions to no-ops

### ISSUE THREAD

**Issue by PaulSnow** [DTRules/DTRules#438]
## Summary

During Excel round-trip testing (#417), discovered that DSL fields containing human-readable text (not valid EL) are being parsed and silently converted to no-ops, breaking rule execution.

## Example

```xml
<action_dsl>Calculate MT tax MFJ</action_dsl>
```

This is not valid EL syntax. The parser logs a warning:
```
Warning: Action 'Calculate MT tax MFJ' is not valid EL, using no-op
```

But then the action does nothing at runtime, causing test failures.

## Impact

- Tests fail with "Stack Check Exception: No result on stack"
- Rules silently do nothing instead of executing
- Users don't realize their rules are broken

## Expected Behavior

Options:
1. **Strict mode**: Fail loudly if DSL field contains invalid EL
2. **Legacy mode**: If DSL looks like a comment (starts with verb like Calculate, Process, Log), treat it as a comment and use the postfix field
3. **Validation**: `dtrules sync check` should report invalid EL as errors, not just warnings

## Reproduction

```bash
cd sampleprojects/TaxReturn
dtrules sync import
go test ./pkg/dtrules/... -run TestTaxReturn
```

## Related

- Part of #417 (Excel round-trip workflow)
- Related to #410 (DSL rename) - the rename made this more visible
